src/ui/MessageInput/utils.ts

```diff
@@ -0,0 +1,23 @@
+// Sanitize that special characters of HTML tags cause XSS issue
+import { BaseChannel } from '@sendbird/chat';
+
+export const sanitizeString = (str?: string) => {
+  return str?.replace(/[\u00A0-\u9999<>]/gim, (i) => ''.concat('&#', String(i.charCodeAt(0)), ';'));
+};
+
+/**
+ * NodeList cannot be used with Array methods
+ * @param {NodeListOf<ChildNode>} childNodes
+ * @returns Array of child nodes
+ */
+export const nodeListToArray = (childNodes?: Node['childNodes'] | null) => {
```

Comment:
why not use type NodeListOf<ChildNode>?

Reply:
I remember there were issues related to TypeScript configuration at the build time, for example, issues like 'set the target to the latest version to use this type'.